app/controllers/miq_request_controller.rb

```diff
@@ -585,22 +588,17 @@ def request_edit_settings(provision_request)
   end
 
   def get_session_data
+    super
     @title        = _("Requests")
     @request_tab  = session[:request_tab] if session[:request_tab]
-    @layout       = layout_from_tab_name(@request_tab)
-    @lastaction   = session[:request_lastaction]
     @showtype     = session[:request_lastaction]
```

Comment:
but you can definitely drop `@showtype`, GenericSessionMixin does handle that.

(And setting it from `request_lastaction` instead of `request_showtype` was definitely a typo, so just dropping the line may actually fix bugs ;).)